Userland/Libraries/LibWeb/CSS/CSSStyleDeclaration.h

```diff
@@ -59,12 +59,12 @@ class PropertyOwningCSSStyleDeclaration : public CSSStyleDeclaration {
     virtual ~PropertyOwningCSSStyleDeclaration() override = default;
 
     virtual size_t length() const override;
-    virtual DeprecatedString item(size_t index) const override;
+    virtual StringView item(size_t index) const override;
```

Comment:
I think it would be better to return `String` in all situations like this, since it allows callers to take ownership of the string. When we return `StringView`, any caller that wants to persist the string is forced to allocate and make a deep copy of it.

Reply:
Right, that makes sense - the reason why I originally didn't do it is because `CSS::string_from_property_id` returns a `StringView`. I figured something may want to call this without needing to make a deep copy / allocate at all, although seems the only caller is in the the prototype, which does a deep copy anyway.

Will make that change :^)

Reply:
Hmm. `CSS::string_from_property_id` could probably return FlyStrings now. I think it only returned StringView to avoid the ErrorOr stuff.